### PR TITLE
Bump @theforeman/vendor

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by evgeni** [comment]
you can't bump a package and its buildrequires in the same PR, so you'll have to split off the foreman change.

also, any reason not to update @theforeman/builder too?

**Comment by xprazak2** [comment]
Thanks, I updating it [separately](https://github.com/theforeman/foreman-packaging/pull/6301), seems like I got a silent failure before and builder did not get updated.

**Review by evgeni — CHANGES_REQUESTED** [review]
please also update theforeman/builder requiremenets

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -259,8 +259,8 @@ BuildRequires: systemd
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 9.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 8.0.0
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec:263]: your math is seriously off

Comment (xprazak2) [reply packages/foreman/foreman/foreman.spec:263]: Maybe I should start drinking coffee.

**Review by evgeni — APPROVED** [review]

**Review by evgeni — CHANGES_REQUESTED** [review]
```
Requires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
Requires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
```
(line 459/460) still needs updating

**Comment by xprazak2** [comment]
Updated.

**Review by evgeni — APPROVED** [review]

**Pull request merged**